rdr_service/dao/genomics_dao.py

```diff
@@ -3813,6 +3813,54 @@ def get_w1il_yes_no_yes_participants(self, start_datetime):
 
             return query.distinct().all()
 
+    def get_results_withdrawn_participants(self):
+        with self.session() as session:
+            array_results = aliased(GenomicSetMember)
+            cvl_results = aliased(GenomicSetMember)
+            return session.query(
+                GenomicSetMember.participantId.label('participant_id'),
+                sqlalchemy.case(
+                    [
+                        (array_results.id.isnot(None), True)
+                    ],
+                    else_=False
+                ).label('array_results'),
+                sqlalchemy.case(
+                    [
+                        (cvl_results.id.isnot(None), True)
+                    ],
+                    else_=False
+                ).label('cvl_results')
+            ).join(
+                ParticipantSummary,
+                ParticipantSummary.participantId == GenomicSetMember.participantId
+            ).outerjoin(
+                array_results,
+                and_(
+                    array_results.id == GenomicSetMember.id,
+                    array_results.gemA1ManifestJobRunId.isnot(None)
+                )
```

Comment:
For my own understanding: would joining the GenomicSetMember table to itself based on the id column essentially join each row to itself?

Reply:
Edited query, good catch.

https://github.com/all-of-us/raw-data-repository/pull/3102/commits/c8282c04ea8cd5a96191f16cf988f415fa9baf70